wiki/Beatmaps/id.md

```diff
@@ -1,78 +1,53 @@
+Sebagaimana yang telah dijelaskan sebelumnya, suatu beatmap yang berstatus Work in Progress dan Pending akan secara otomatis dipindahkan ke dalam [Graveyard](#graveyard) apabila beatmap yang bersangkutan tidak diperbaharui selama 4 minggu.
 
 ### Qualified
 
-Beatmap berstatus Qualified menggunakan ikon centang (![Check icon](/wiki/shared/status/qualified.png)) di halaman pemilihan lagu.
-
-Beatmap ini berada di tahap terakhir dari proses beatmapping dan modding. Sebelum beatmap mulai, dialog akan ditampilkan, memperlihatkan status dari beatmap akan menyebabkan:
-
-- Tidak ada [pp](/wiki/pp) yang akan diberikan.
-- Semua skor akan dihapus saat status beatmap berubah.
-  - Jika status beatmap berubah menjadi ranked, approved, atau kembali ke pending; semua skor akan dihapus!
-
-Ketika sebuah beatmap berstatus qualified, beatmap telah memenuhi [kriteria ranking](/wiki/ranking_criteria) dan telah diperiksa oleh dua [Beatmap Nominators](/wiki/Beatmap_Nominators). Beatmap yang berstatus qualified memasuki tahap terakhir pertahanan karena mereka harus berada dalam status ini selama satu minggu sebelum menjadi [ranked](#ranked) atau [approved](#approved) secara permanen.
-
-Namun, selama durasi satu minggu, [QATs](/wiki/QAT) diizinkan untuk mencari permasalahan pada beatmap ini. Jika hal ini terjadi, status beatmapset akan kembali ke pending dan prosesnya akan diulangi lagi.
-
-### Loved
-
-Beatmap berstatus Loved menggunakan ikon hati (![Heart icon](/wiki/shared/status/loved.png)) di halaman pemilihan lagu.
-
-Beatmap ini mungkin tidak memenuhi [kriteria ranking](/wiki/ranking_criteria) tetapi *dicintai* oleh komunitas dan secara teratur ditambahkan berdasarkan suara komunitas. Sebelum beatmap mulai, dialog akan ditampilkan, memperlihatkan status dari beatmap akan menyebabkan:
-
-- Tidak ada [pp](/wiki/pp) yang akan diberikan.
-- Semua skor akan dihapus saat pembuat memperbarui beatmapnya.
-
-Status ini akan memberikan leaderboard pada beatmap
-
-Untuk menjadikan sebuah beatmap berstatus loved, lihat kiriman forum: [Get your beatmap Loved!](https://osu.ppy.sh/community/forums/topics/549835).
+Qualified merupakan status beatmap yang disematkan kepada beatmap-beatmap yang tengah berada dalam tahapan akhir [prosedur ranking beatmap](/wiki/Beatmap_ranking_procedure). Beatmap-beatmap ini sebelumnya telah dinominasikan oleh setidaknya dua orang anggota [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) yang telah meninjau dan menilai kelayakan kualitas beatmap-beatmap tersebut berdasarkan [Ranking Criteria](/wiki/Ranking_Criteria). Setiap beatmap yang masuk ke dalam kategori Qualified akan ditempatkan di dalam [antrian ranking](/wiki/Beatmap_ranking_procedure/Ranking_queue), di mana beatmap-beatmap tersebut akan secara otomatis berpindah ke dalam kategori [Ranked](#ranked) apabila setelah sekian waktu beatmap tersebut tidak [didiskualifikasi](/wiki/Beatmap_ranking_procedure#nomination-resets) karena satu dan lain hal.
```

Comment:
one sentence might be missing: `This period serves as a last line of defense in which any flaws and mistakes that might have slipped through can be fixed.`

Reply:
will take a look at this a bit later